File: app/src/main/java/de/tum/in/tumcampusapp/models/tumcabe/RoomFinderMap.java
package de.tum.in.tumcampusapp.models.tumcabe;

/**
 * This class is used as a model for maps in Roomfinder retrofit request.
 */
public class RoomFinderMap {
    String map_id;
    String description;

    public RoomFinderMap(String map_id, String description) {
        this.map_id = map_id;
        this.description = description;
    }

    public String getMap_id() {
        return map_id;
    }

    public String getDescription() {
        return description;
    }
}

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/roomfinder/model/RoomFinderMap.kt
package de.tum.`in`.tumcampusapp.component.tumui.roomfinder.model

/**
 * This class is used as a model for maps in Roomfinder retrofit request.
 */
@Deprecated("""Please use NavigationDetailsDto instead""")
data class RoomFinderMap(
    var map_id: String = "",
    var description: String = ""
)
